packages/core/src/libraries/hook/index.ts

```diff
@@ -15,6 +15,10 @@ import type Provider from 'oidc-provider';
 
 import RequestError from '#src/errors/RequestError/index.js';
 import { LogEntry } from '#src/middleware/koa-audit-log.js';
+import {
+  type InteractionHookContext,
+  type InteractionHookResults,
+} from '#src/routes/interaction/middleware/koa-interaction-hooks.js';
```

Comment:
i think you are still having the circular dependency here

Reply:
updated.